CondCore/Utilities/python/tier0.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, message):
 
 def unique(seq, keepstr=True):
     t = type(seq)
-    if t in (unicode, str):
+    if t is str:
```

Comment:
As far as I understand `unicode` here was removed to avoid the error
```
NameError: name 'unicode' is not defined
```
But what was its purpose here at first? Is it used somewhere?
How is this tier0.py used, besides the test that you suggest in the PR description?

Reply:
@perrotta in Python2, there were two string types: `str` and `unicode`. They were merged together in Python3.

Reply:
Exactly, thanks Iarspider for the answer.

Answering the other question: `Tier0Handler` is used here https://github.com/cms-sw/cmssw/blob/a08cd1123d9fa59ed609156d678c6dc760ccc9c9/CondCore/Utilities/scripts/conddb#L713

and regarding this `unique` function with removed `unicode`: it's used only in `getGlobalTag` and the test in this file is quite complete for this function because it tests all the 2 used argument values (`'reco_config'` and `'express_config'`) so I don't think there's any use case that could surprise us